pkg/agent/util/ethtool/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package ethtool provides functions that make antrea to run on Kind.
```

Comment:
```suggestion
// Package ethtool provides Go wrappers for ioctl ethtool system calls on Linux
```

Let's not make this about Kind here (not to mention that the root cause is not really Kind but the netdev OVS datapath type).